logger/src/lib.rs

```diff
@@ -45,6 +45,7 @@ where
     }
 
     async fn get_logs(&self, deployment_id: String) -> Result<Vec<LogLine>, Error> {
+        tracing::info!(deployment_id = %deployment_id, "fetching logs for deployment");
```

Comment:
I think we will be fine removing all the info traces in this PR

Reply:
Hmm, will we see anything if there are no events?

Reply:
Ah, we will see it in Honeycomb, but not in the logger itself. I'll remove the events then, since they are quite verbose.